recipe/recipe.yaml

```diff
@@ -14,7 +14,7 @@ source:
     - fix-deps.patch
 
 build:
-  number: 0
+  number: 1
```

Comment:
Bumping the build number seems unnecessary to me. I don't see any reason to upload new binaries for this feedstock infrastructure update. No pins in `.ci_support/` were updated after the rerendering, so no binaries will be uploaded if we keep the build number at 0.